tools/lkl/lib/hijack/hijack.c

```diff
@@ -222,6 +229,10 @@ int ioctl(int fd, unsigned long req, ...)
 }
 
 
+#if defined(__ANDROID__) && __LKL__BITS_PER_LONG == 32
+#define __lkl__NR_fcntl __lkl__NR_fcntl64
```

Comment:
We can move this to tools/lkl/include/lkl.h.

Reply:
agree.  will fix this.